src/main.rs

```diff
@@ -1,17 +1,83 @@
-//! Program to take a Brainfuck program at the specified file path and 'run' it
+//! Brainfuck Interpreter. This crate takes created a [BfProgram] from a file and runs it on a
```

Comment:
DRYness

```suggestion
//! Brainfuck Interpreter. This takes created a [BfProgram] from a file and runs it on a
```

Also, have you checked that the `[BfProgram]` actually makes a link, or does it need to have backticks around `BfProgram` ??